.github/workflows/deploy_api.yml

```diff
@@ -41,7 +148,60 @@ jobs:
         path: covid-data-public
         lfs: true
         ref: '${{env.COVID_DATA_PUBLIC_REF}}'
+    - name: Setup Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.7.6'
+        architecture: 'x64'
+    - name: Cache Pip
+      uses: actions/cache@v1
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('requirements.txt') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-
+          ${{ runner.os }}-
+    - name: Install Dependencies
+      working-directory: ./covid-data-model
+      run: pip install -r requirements.txt
+
+    # - uses: actions/download-artifact@v1
+    #   with:
+    #     name: model-results
+    #     path: ./api-results
+    - name: Build DoD (run.sh .. .. execute_summaries)
+      env:
+        COVID_MODEL_CORES: 96
+      run: ./covid-data-model/run.sh ./covid-data-public /data/api-results-${{env.SNAPSHOT_ID}} execute_dod
+    # - name: Upload DoD with Model Results
```

Comment:
Why are these commented out?

Reply:
for future debugging, but maybe worth cleaning up later